csharp/ql/src/Telemetry/ExtractorInformation.ql

```diff
@@ -0,0 +1,195 @@
+/**
+ * @name C# extraction information
+ * @description Information about the extraction for a C# database
+ * @kind metric
+ * @tags summary telemetry
+ * @id cs/telemetry/extraction-information
+ */
+
+import csharp
+import semmle.code.csharp.commons.Diagnostics
+
+predicate fileCount(string key, int value) {
+  key = "Number of files" and
+  value = strictcount(File f)
```

Comment:
Could it make sense to report this if it is also 0? (use `count` instead of `strictcount`).

Reply:
I don't think that can ever happen, as there should in those cases be no TRAP, and hence no DB to begin with.